rpc/client_interceptors.go

```diff
@@ -45,3 +47,21 @@ func contextWithSpanMetadata(ctx context.Context) context.Context {
 	)
 	return ctx
 }
+
+// UnaryClientReauthInterceptor clears the access token stored on creds in the
```

Comment:
ReauthInterceptor implies that it actively reauths for the caller, which this doesn't, so I'd like a different name - trying to think of other still succinct names but not succeeding.

maybe `ClearInvalidAuthInterceptor`? or even just `AuthInterceptor`?

Reply:
Yeah good point; we're not doing any active re-authing here. Switched to `UnaryClientInvalidAuthInterceptor`.